src/acl/group.test.ts

```diff
@@ -0,0 +1,1158 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { dataset } from "@rdfjs/dataset";
+import {
+  LitDataset,
+  WithResourceInfo,
+  IriString,
+  unstable_Access,
+  unstable_AclDataset,
+  unstable_WithAcl,
+  WebId,
+} from "../interfaces";
+import { DataFactory } from "../rdfjs";
+import {
+  unstable_getGroupDefaultAccessOne,
+  unstable_getGroupResourceAccessOne,
+  unstable_getGroupResourceAccessAll,
+  unstable_getGroupDefaultAccessAll,
+  unstable_getGroupAccessOne,
+  unstable_getGroupAccessAll,
+} from "./group";
+
+function addAclRuleQuads(
+  aclDataset: LitDataset & WithResourceInfo,
+  group: WebId,
```

Comment:
I'm not sure about this, but I don't think we call a Group's IRI a WebID? Or do we?

```suggestion
  group: IriString,
```

Reply:
We don't, I tried to be careful about this but missed this one, good catch :)

Reply:
And also I realized that mid-dev, which is why there are some blatant errors such as `https://some.pod/group#webId` ^^